gulp e2e task: Error: Spec patterns did not match any files

<Have you checked issues section which may have your similar issue?>

<Please take your time filling these required information for posting issue>

## Issue Overview
<Please explain the issue overview here, short but concise and to-the-point>

running "gulp e2e" throws: Error: Spec patterns did not match any files

the file task/test.js appears to have the wrong path to the transpiled spec files.

line 23:
  gulp.src(config.e2e + '**/*.spec.js')

"gulp e2e" works if line 23 is changed to point to the files under tmp:

gulp.src(config.tmp + config.e2e + '**/*.spec.js')

## Issue Description
<Please explain the issue in details>

When running "gulp e2e" after launching "gulp serve-dev" the following error is thrown:
$ gulp e2e
(node:27963) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
[01:32:26] Using gulpfile ~/Documents/repos/angular2-starter/gulpfile.js
[01:32:26] Starting 'driver-update'...
[01:32:26] Starting 'clean-ts-test'...
[01:32:26] Finished 'clean-ts-test' after 17 ms
[01:32:26] Starting 'tsc-e2e'...
selenium standalone is up to date.
chromedriver is up to date.
[01:32:26] Finished 'driver-update' after 228 ms
[01:32:28] Finished 'tsc-e2e' after 1.72 s
[01:32:28] Starting 'e2e-test'...
[01:32:28] Finished 'e2e-test' after 1.83 ms
[01:32:28] Starting 'e2e'...
[01:32:28] Finished 'e2e' after 1.95 μs
[launcher] Process exited with error code 1
/Users/foo/Documents/repos/angular2-starter/node_modules/q/q.js:155
                throw e;
                ^

Error: Spec patterns did not match any files.
    at Runner.run (/Users/foo/Documents/repos/angular2-starter/node_modules/protractor/built/runner.js:286:11)
    at TaskRunner.run (/Users/foo/Documents/repos/angular2-starter/node_modules/protractor/built/taskRunner.js:109:27)
    at createNextTaskRunner (/Users/foo/Documents/repos/angular2-starter/node_modules/protractor/built/launcher.js:216:20)
    at /Users/foo/Documents/repos/angular2-starter/node_modules/protractor/built/launcher.js:239:7
    at _fulfilled (/Users/foo/Documents/repos/angular2-starter/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/foo/Documents/repos/angular2-starter/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/foo/Documents/repos/angular2-starter/node_modules/q/q.js:796:13)
    at /Users/foo/Documents/repos/angular2-starter/node_modules/q/q.js:604:Error running E2E testing
## Reproducables
<Please explain the way to reproduce your issue, or even better with Plunker link>

clone repo, run "gulp serve-dev", then in another terminal "gulp e2e"

## Information
|                      	|                                 	|
|---------------------	|---------------------------------	|
| **Operating System** |	OSX 	|
| **Node version**     	| 6.1.0   	|
| **NPM Version**      	| 3.8.9    	|
| **Environment**       | Browser          |

--- a/tasks/test.js
+++ b/tasks/test.js
@@ -20,7 +20,7 @@ gulp.task('unit-test', ['tsc'], function (done) {
 gulp.task('e2e', ['e2e-test']);
 gulp.task('driver-update', gulpProtractor['webdriver_update']);
 gulp.task('e2e-test', ['driver-update', 'tsc-e2e'], function () {
-    gulp.src(config.e2e + '**/*.spec.js')
+    gulp.src(config.tmp + config.e2e + '**/*.spec.js')
     .pipe(gulpProtractor.protractor({
         configFile: 'protractor.conf.js',
         args: ['--baseUrl', config.e2eConfig.seleniumTarget]

